src/olympia/devhub/templates/devhub/addons/listing/item_actions_theme.html

```diff
@@ -4,7 +4,7 @@ <h5>{{ _('Actions') }}</h5>
     <li>
       <a href="{{ addon.get_dev_url() }}" class="tooltip"
          title="{{ _("Edit details about this theme's listing.") }}">
```

Comment:
Can you update the tooltip here too, please?

Reply:
https://github.com/mozilla/addons-server/pull/9924/commits/dc25ec385d52cde50abd032d068270f3e63196b7